components/widgets/ui/src/main/resources/PhenoTips/GeneVariantMacros.xml

```diff
@@ -0,0 +1,1420 @@
+#end
+##
+##
+#macro(__variant_head $count $geneObjNumber $geneName $classname $options)##
+|(%class="pseudoindent"%) |=(% colspan="4" class="variant variant-title gene-$geneObjNumber" %) $services.localization.render("${classname}.variantTable.title", [$geneName, $count]) \\
+(%class="variant-gene-$geneObjNumber variant-title-row variant-hide-heading-$geneObjNumber"%)|(%class="pseudoindent"%) |=(% class="variant var-row-number"  %) #(%%)#foreach($propName in $options.defaultProperties) |=(% class="variant col-label" %) $services.localization.render("${classname}.variantTable.${propName}") #end |=(% class="transparent-row" %) 
+#end
+##
+##
+##
+#macro (__variant_row_empty $variantClassName $geneObjNumber $options)
+#set($fakeObj = $doc.newObject($variantClassName))
+#set($variantIndex = 'VARIANT_INDEX_PLACEHOLDER')
+#set($geneObjNumber = 'GENE_INDEX_PLACEHOLDER')
+#set($count = 'VRCOUNT_PLACEHOLDER')
+|(% class="v-collapsed" %) $doc.display('genesymbol', $options.mode, $fakeObj)|(% class="pseudoindent" %) |(% class="variant-row-count variant-${variantIndex} variant" %)$count#foreach($propName in $options.defaultProperties)|(% class="variant variant-default-input $propName variant-$variantIndex" %) $doc.display($propName, $options.mode, $fakeObj) #end#__variant__deleteTool($variantObj)\\
+(% class="variant-gene-$geneObjNumber variant-moreinfo-row v-collapsed variant-hide-heading-$geneObjNumber" %)|(% class="pseudoindent" %)|(% class = "variant" %)|(% colspan="3" class="variant moreinfo" %)(% class="variant-moreinfo-editbutton-row variant-$variantIndex" %)((( #foreach ($prop in $fakeObj.xWikiClass.properties) #if ($prop.getName() != 'genesymbol' &amp;&amp; !$options.defaultProperties.contains($prop.getName()))
+  
+#set ($isInputString = $options.inputStrings.contains($prop.getName())) 
+#set($rawValue = "")
+|(% class="variant-moreinfo variant-property-name-$variantIndex %)**${prop.translatedPrettyName}**| (% class="variant-property-value ${prop.getName()}-$variantIndex variant-label-$variantIndex" %) #if($isInputString) $rawValue #else $doc.display($prop.getName(), 'view', $fakeObj )#end (%%) (% class="v-collapsed variant-input-$variantIndex" %)$doc.display($prop.getName(), $options.mode, $fakeObj )#end##
+#end
+))) (% class="variant-moreinfo-editdonebutton-row variant-$variantIndex" %)\\
+#end
+##
+##
+##
+#macro (__variant_row $variantObj $count $variantIndex $geneObjNumber $options)
+#if ($options.mode == 'edit')|(% class="v-collapsed" %) $doc.display('genesymbol', $options.mode, $variantObj)#end|(% class="pseudoindent" %) |(% class="variant-row-count variant-${variantIndex} variant" %)$count#foreach($propName in $options.defaultProperties)|(% class="variant variant-default-input $propName variant-$variantIndex" %)$doc.display($propName, $options.mode, $variantObj)#end#if ($options.mode == 'edit')#__variant__deleteTool($variantObj)#end\\
```

Comment:
are there any guidelines how to safely break up lines here and not generate anything new?